app/tools/android/make_apk.py

```diff
@@ -690,6 +691,13 @@ def MakeEmbeddedApk(options, app_info, app_dir, packaged_archs):
       sys.exit(13)
 
 
+@atexit.register
+def PrintDeprecationMessage():
+    print('\n\n\n   WARNING!!! This command is deprecated and will be removed soon.\n\n'
```

Comment:
Nitpick 3: maybe say "make_apk" or "The make_apk tool" instead of "this command"?